jsonnet_test.go

```diff
@@ -327,3 +328,23 @@ func TestTLATypes(t *testing.T) {
 	}
 	assertVarOutput(t, jsonStr)
 }
+
+func TestSetTraceOut(t *testing.T) {
+	traceOut := &strings.Builder{}
+	vm := MakeVM()
+	vm.SetTraceOut(traceOut)
+
+	const msg = "Some Trace Message"
+	expected := fmt.Sprintf("TRACE: :1 %s", msg)
```

Comment:
Not sure why the file name "blah.jsonnet" does not show up in the message.

Reply:
I've added a comment above on how to fix this.